src/components/AddPaymentMethodMenu.tsx

```diff
@@ -71,13 +73,13 @@ function AddPaymentMethodMenu({
 
     // We temporarily disabled P2P debit cards so we will automatically select the personal bank account option if there is no other option to select.
     useEffect(() => {
-        if (!isVisible || !isPersonalOnlyOption) {
+        if (!isVisible || !isPersonalOnlyOption || introSelectedStatus.status === 'loading') {
```

Comment:
can use reusable `isLoadingOnyxValue` to avoid magic strings throughout the repo
```suggestion
        if (!isVisible || !isPersonalOnlyOption || isLoadingOnyxValue(introSelectedStatus)) {
```